lib/gstreamer/va/encoder.py

```diff
@@ -96,7 +96,11 @@ class EncoderTest(BaseEncoderTest):
 
   def before(self):
     super().before()
-    os.environ["GST_VA_DRM_DEVICE"] = get_media().render_device
+    renderDevice = get_media().render_device.split('/')[-1]
+    if renderDevice != 'renderD128':
+      vars(self).update(
+        gstencoder= vars(self).get('gstencoder').replace('va', f'va{renderDevice}')
+      )
```

Comment:
I prefer the renderDevice check be done in the Encoder class, `gstencoder` property... not the EncoderTest class.